sorc/ocnice_prep.fd/ocniceprep.F90

```diff
@@ -121,6 +125,25 @@ program ocniceprep
   call getfield(trim(gridfile),  'depth', dims=(/nxr,nyr/), field=bathydst)
   call nf90_err(nf90_close(ncid), 'close: '//trim(gridfile))
 
+  if (.not. do_ocnprep) then
+  ! -----------------------------------------------------------------------------
+  ! obtain the land mask for the ice model on the source grid to QC the ice fields
+  ! -----------------------------------------------------------------------------
+
+     allocate(a2d(nxt,nyt)); a2d = 0
+     allocate(kmt(nxt*nyt)); kmt = 0
+
```

Comment:
Are `a2d` and `kmt` big arrays? It looks like `a2d` can be deallocated before the 'endif'.  `kmt` is used below, so you may need to keep it in memory?

Reply:
Yes, I had noticed that also, and forgot to fix. They're not huge (1440x1080), but I will add the deallocate do that. Thanks.